### PR TITLE
Fix typo in README

### PR DESCRIPTION
**Pull request by morgandonze**

### CODE REVIEW

**Review by derekgreenberg — CHANGES_REQUESTED** [review]
change spelling of `recomended` to ` recommended`

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@
 | 2017 | Andross (recommended)  | React Navigation, Redux, Redux Sagas          | https://github.com/infinitered/ignite-ir-boilerplate-andross  |
 | 2016 | Ignite 1.0 | React Native Router Flux, Redux, Redux Sagas | https://github.com/infinitered/ignite-ir-boilerplate-2016 |
 
-To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently reccomended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
+To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently recomended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
```
Comment (kevinvangelder) [inline README.md]:
```suggestion
To support legacy versions of Ignite, https://www.npmjs.com/package/ignite-ir-boilerplate will continue to deliver our currently recommended boilerplate, but we recommend updating Ignite CLI so you can manually select which boilerplate you would like to use.
```

**Review by kevinvangelder — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (derekgreenberg) [reply README.md]: Be sure to spell "recomended" as "recommended" in the above change request.

**Comment by morgandonze** [comment]
Fixed. Github still says merging is blocked until requested changes are addressed, but they have been addressed. Their code change recommendations feature is still in beta after all, I guess.

**Review by kevinvangelder — APPROVED** [review]

**Review by derekgreenberg — APPROVED** [review]

**Pull request merged**